electrum/wallet.py

```diff
@@ -1840,7 +1835,7 @@ def make_unsigned_transaction(
             coins: Sequence[PartialTxInput],
```

Comment:
changes needed, ~13 lines above. `config.estimate_fee` does not exist anymore:
```py
fee = partial(self.config.estimate_fee, allow_fallback_to_static_rates=True) # to avoid NoDynamicFeeEstimates
```
(I cannot comment directly on that line as it is too far away from any diff)